### ISSUE TITLE
(WIP) Starting ToonzFarm controller on mac

### ISSUE THREAD

**Issue by flaviohr** [opentoonz/opentoonz#2409]
### Question
The last nightly version of OpenToonz can't start on mac high sierra.

### Screenshot or Video Reference
![screen shot 2018-12-09 at 7 00 22 pm](https://user-images.githubusercontent.com/19403672/49704251-0d107680-fbe7-11e8-993f-606f9e24aa1e.png)
![screen shot 2018-12-09 at 7 03 03 pm](https://user-images.githubusercontent.com/19403672/49704253-1994cf00-fbe7-11e8-9eee-521a4dfdb9a5.png)
I can't start the controller.
![screen shot 2018-12-09 at 7 19 24 pm](https://user-images.githubusercontent.com/19403672/49704279-64164b80-fbe7-11e8-97b9-5c825a56a3b0.png)

### System Information
* **OpenToonz Version**: 1.2.1 nightly version
* **Operating System**: high sierra 10.13.6
* **CPU**: intel core i5 2.3 GHz 
* **Memory**: 16 GB RAM
* **Graphics Card**:  Intel Iris Plus Graphics 640 1536 MB
* **Graphics Tablet**:  Wacom Bamboo CTL 460

**Comment by artisteacher** [comment]
Was this working in the previous nightly build? I’d like to try setting up the farm myself, but I’m still trying to understand the instructions in the manual...

**Comment by flaviohr** [comment]
Basically, The farm will setting up making these folders on Opentoonz Stuff Folder:
/Applications/OpenToonz/OpenToonz_1.2_stuff/toonzfarm/
and then:
![screen shot 2018-12-10 at 12 21 32 am](https://user-images.githubusercontent.com/19403672/49710764-ec114b00-fc11-11e8-911d-0698abeabe4e.png)
and in "config" Folder:
![screen shot 2018-12-10 at 12 21 50 am](https://user-images.githubusercontent.com/19403672/49710773-fcc1c100-fc11-11e8-9a66-38649465bd06.png)
on "controller.txt" type in the same roww, the host, IP, and a port. something like: 
your-user-macosx.local 192.168.0.1 1000
and in "servers.txt" same as above, but in the rows below, you will put another computers, like:
your-user-macosx.local 192.168.0.1 1000
another-user.local 192.168.0.2 1000
another-example.local 192.168.0.3 1000

The "toonzfarm" folder will be now the ROOTFARM folder, but before you will create anothet txt called "configfarmroot.txt" in: 
/Applications/OpenToonz_1.2/Contents/Resources/
with the ROOTFARM above (/Applications/OpenToonz/OpenToonz_1.2_stuff/toonzfarm/) where "config", "data", and "logs" folder are inside.

And, in OT/ farm tab, in Farm Global Root: put the local of the ROOTFARM folder. (/Applications/OpenToonz/OpenToonz_1.2_stuff/toonzfarm/)

But... a message in Message Center, say "Please, Start the controller before".

I think this is all I need to do, because I read the Toonz Premium manual and I that tested. but OT still not working on mac.

**Comment by artisteacher** [comment]
@flaviohr - thanks for the explanation with screenshots. That is definitely helpful - especially when combined with @damies13's instructions from #1154. It seems the updated instructions never made it to the OpenToonz Docs. I think they are much clearer than the instructions that are currently posted.

**Comment by RodneyBaker** [comment]
Any update here will likely occur in 2019 development cycle.

**Comment by flaviohr** [comment]
I think this will need more attention for large production, but I don't tested that function on windows or linux.

**Comment by flaviohr** [comment]
@RodneyBaker no, I saw some people like this guy https://www.youtube.com/watch?v=NYgvFi7auQE using the toonzfarm on windows.
well, I tried to using on mac, but all configuration needed was not efficient enough, because when I started the render on farm tab, the controller are not active. like the second image on top of this topic.

I need to use the render farm, because I have a lot of animations, but when I render using the output settings, I notice when my mac is with no ram enough the render fails sometimes and crash OT (nightly and stable versions), (I think this will be a bug with memory allocation or something related. but I think this will need a new topic for that possible bug and I need more info about to do that
